client/src/components/Chat.jsx

```diff
@@ -215,7 +215,7 @@ const Chat = () => {
 
         socket.emit(NEW_EVENT_TYPING, { chatId: app.currentChatId, isTyping: false });
         const d = new Date();
-        let message = inputRef.current.value;
+        let message = inputRef.current.value.trim();        // Trim the message to remove the extra spaces
```

Comment:
not a good way to fix it, check if the message is empty and then do a return

Reply:
That  is already checked in the if condition

Reply:
Check Line 255

Reply:
Hey Kindly Check line No 255 I think there is no need to check the condition again

Reply:
then your pr should be fixing why it's not working instead of this

Reply:
which means it should check if it's empty spaces, but i guess that fixes it

Reply:
> not a good way to fix it, check if the message is empty and then do a return

Hey Kindly Check line No 255 I think there is no need to check the condition again it is already present in the if condition